### PR TITLE
enhancement: change airdrop tile for multiple periods

### LINKED ISSUE #2759: [Task]: Modify design of `StakingAirdrop` component for multiple periods
### Task description

Now that we have multiple periods we need to match the implementation to the new designs, so that we can show the total rewards for all periods and the new rewards for current period.

### Requirements

- [ ] Add total airdrop reward to tile
- [ ] Only show the current rewards and duration if there is a period on going
- [ ] Only show staking indicator if the event is in commencing or holding

### Acceptance criteria

PR approved and merge that achieves above requirements.

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary
The `StakingAirdrop` components needs to accommodate for multiple staking periods.

### Changelog
```
- Separate event state into two separate stores for Assembly and Shimmer events
- Modify airdrop tile to display current and total rewards if event is ongoing
- Create two stores for total airdrop rewards
- Change to test events on the devnet
```

## Relevant Issues
- Closes #2759

## Type of Change
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [x] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
- __Desktop__
	- [ ] MacOS
	- [x] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by nicole-obrien — CHANGES_REQUESTED** [review]
Getting this error sending a transaction

File: packages/shared/components/popups/Transaction.svelte [hunk 1]
```diff
@@ -8,7 +8,11 @@
     import { AvailableExchangeRates, CurrencyTypes } from 'shared/lib/typings/currency'
     import { Locale } from '@core/i18n'
     import { formatUnitBestMatch, formatUnitPrecision } from 'shared/lib/units'
-    import { selectedAccountParticipationOverview, stakingEventState } from 'shared/lib/participation/stores'
```
Comment (begonaalvarezd) [inline packages/shared/components/popups/Transaction.svelte:11]: `stakingEventState` can not be removed because its used in line 51

**Review by nicole-obrien — CHANGES_REQUESTED** [review]
Nearly there, just a few comments 🌆

**Review by nicole-obrien — APPROVED** [review]
🚀 🚀 🚀

**Pull request merged**